mono/utils/mono-context.h

```diff
@@ -33,6 +33,7 @@ typedef struct __darwin_xmm_reg MonoContextSimdReg;
 typedef struct _libc_xmmreg MonoContextSimdReg;
 #elif defined(HOST_WIN32)
 #define MONO_HAVE_SIMD_REG
+//#define MONO_HAVE_SIMD_REG_AVX
 #include <emmintrin.h>
```

Comment:
Is this set somewhere ?

Reply:
No not at the moment in upstream, was thinking that we should have it as a ENABLE_XXX define or set in configure step, but since it is not enabled in upstream (since I guess we can't take a dependencies on AVX at the moment), I just left it together with the current define enabling XMM support and that is not something that can be configured either at the moment.